mixlib-versioning.gemspec

```diff
@@ -13,16 +13,10 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/chef/mixlib-versioning"
   spec.license       = "Apache 2.0"
 
-  spec.required_ruby_version = ">= 2.2"
+  spec.required_ruby_version = '>= 1.9'
```

Comment:
We should add a comment here on why we support such an old version of Ruby. Otherwise we'll probably forget and do this same thing again since it seems like the right thing to do